testing/octave/APKBUILD

```diff
@@ -1,31 +1,30 @@
 # Contributor: Daniel Sabogal <dsabogalcc@gmail.com>
 # Maintainer:
 pkgname=octave
-pkgver=4.2.0
-pkgrel=1
+pkgver=4.2.2
+pkgrel=0
 pkgdesc="A high-level language, primarily intended for numerical computations"
 url="https://www.gnu.org/software/octave/"
 arch="all !s390x"
 license="GPL-3.0"
-depends=""
 subpackages="$pkgname-doc"
-makedepends="autoconf automake bison curl fftw-dev fltk-dev fontconfig-dev
-	freetype-dev gfortran glu-dev gnuplot imagemagick-dev hdf5-dev
-	lapack-dev libsm-dev libltdl lcms2-dev pcre-dev perl pkgconf qt-dev
-	readline-dev texinfo zlib-dev"
-source="ftp://ftp.gnu.org/gnu/octave/$pkgname-$pkgver.tar.gz
+makedepends="autoconf automake libtool bison curl-dev fftw-dev flex fltk-dev fontconfig-dev
+	freetype-dev gfortran ghostscript-dev glu-dev gnuplot graphicsmagick-dev hdf5-dev
+	icoutils lapack-dev lcms2-dev libltdl librsvg-dev libsm-dev libsndfile-dev pcre-dev
+	perl pkgconf portaudio-dev qt-dev readline-dev texinfo texlive-full zlib-dev"
+source="https://ftp.gnu.org/gnu/octave/$pkgname-$pkgver.tar.gz
 	less-docs.patch
 	libinterp-deps.patch
 	abs.patch
 	gnulib.patch"
-builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
`$buildir` is used below (lines 26|24, -|46 and -|51). Shouldn't this be kept?

Reply:
Since it is the default path then abuilder does not need it to be declared.